apps/src/netsim/NetSimPacketSizeControl.js

```diff
@@ -0,0 +1,108 @@
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+
+ maxlen: 90,
+ maxstatements: 200
+ */
+/* global $ */
+'use strict';
```

Comment:
We should disable this warning, at least in apps; all of our files get function-scoped when they are compiled, thus this already is the function form of "use strict".